src/test/java/com/rocana/lucene/codec/v1/TestRocanaSearchCodecV1.java

```diff
@@ -113,31 +106,14 @@ public void getActualPostingsFormatShouldReturnPostingsFormatFromSPI() {
   }
 
   /**
-   * This test ensures we don't rename the codec class and forget to
-   * update the SPI config file.
-   *
-   * In META-INF/services/org.apache.lucene.codecs.Codec we store
-   * the fully qualified codec name. If we rename the Codec class
-   * we have to update the SPI config file too. If we forget, this
-   * test reminds us by failing.
+   * This test ensures we registered our codec properly in the SPI file:
+   * META-INF/services/org.apache.lucene.codecs.Codec
    */
   @Test
   public void validateJavaServiceProviderConfigFile() throws Exception {
-    File serviceProviderConfigFile = findCodecServiceProviderFile();
-    Properties properties = new Properties();
-
-    try (InputStream inputStream = new FileInputStream(serviceProviderConfigFile)) {
-      properties.load(inputStream);
-    }
+    Codec codec = Codec.forName(RocanaSearchCodecV1.SHORT_NAME);
```

Comment:
Yep! This was @joey's idea and it's a good one. Rather than manually parsing the file we should just let Lucene load it up like normal and we can verify by doing the SPI (Service Provider Interface) lookup, which is `Codec.forName(..)`.

Joey's idea also works if we add a RoanaSearchCodecV2 and have both RoanaSearchCodecV2 and RoanaSearchCodecV1 in this file at the same time. My previous impl would have failed with that.